interface.go

```diff
@@ -18,28 +17,11 @@ type HyperBlockProcessor interface {
 	Process(hyperBlock *hyperBlock.HyperBlock) (*schema.BlockResult, error)
 }
 
-type Driver interface {
-	SaveBlock(args *indexer.ArgsSaveBlockData) error
-	RevertIndexedBlock(header data.HeaderHandler, body data.BodyHandler) error
-	SaveRoundsInfo(roundsInfos []*indexer.RoundInfo) error
-	SaveValidatorsPubKeys(validatorsPubKeys map[uint32][][]byte, epoch uint32) error
-	SaveValidatorsRating(indexID string, infoRating []*indexer.ValidatorRatingInfo) error
-	SaveAccounts(blockTimestamp uint64, acc []data.UserAccountHandler) error
-	FinalizedBlock(headerHash []byte) error
-	Close() error
-	IsInterfaceNil() bool
-}
-
 type AccountsAdapter interface {
```

Comment:
is this still needed?

Reply:
No, but will not delete it now.
Each subcomp (e.g. accounts, block, etc. processor) will be deleted with its associated interfaces, once it will be replaced.
So, baby steps for each new processor